lib/cost_report.rb

```diff
@@ -24,23 +98,56 @@ def matching_clusters
     end
   end
 
-  def add_ht_item_to_freq_table(ht_item)
-    overlap = HtItemOverlap.new(ht_item)
-    overlap.matching_orgs.each do |organization|
-      @freq_table[organization.to_sym][overlap.matching_orgs.count] += 1
+  def total_hscore(member)
+    spm_total(member.to_sym) + mpm_total(member.to_sym) + ser_total(member.to_sym)
+  end
+
+  [:spm, :ser, :mpm].each do |format|
+    # HScore for a particular format
+    define_method "#{format}_total".to_sym do |member|
+      total = 0.0
+      freq_table[member.to_sym][format].each do |num_orgs, freq|
+        total += (1.0 / num_orgs * freq)
```

Comment:
I had to read this three times. Why not just `total += freq.to_f / num_orgs`?

Reply:
Definition in my head is sum of hshares for n items. 6 of one, half dozen of the other. Will simplify.